Create SECURITY.md

gabriela-ech opened the pull request:
Security Policy redaction

gabriela-ech committed:
Create SECURITY.md

Security Policy redaction

HR2590 commented on SECURITY.md:
Remove :white_check_mark:

gabriela-ech committed:
Update SECURITY.md

It was updated the line and was changed into a comment that has a description of the version.

HR2590 reviewed (approved):
It's looks better thanks!!
